packages/transactions/src/OutboundShipment/DetailView/DetailView.test.tsx

```diff
@@ -1,23 +1,85 @@
 import React from 'react';
 import { Route } from 'react-router';
 import { render, waitFor } from '@testing-library/react';
-
+import AppBar from '../../../../host/src/AppBar';
```

Comment:
Ew!

I had a lot of thoughts about this..

- `TestingAppBar` .. ?
- Thoughts around component libraries..
- If we want a consistent look and feel to web apps, we likely want the `AppBar` in `common`, in at least, some form.
- We could go further and have an `AppLayout` component, with a drawer and `AppBar` (And RHS Drawer..?)
- Since we the components are pretty tightly coupled to the translation module.. and also there are a bunch of other providers that we're coupled to right now, we might also want them in the `AppLayout`..
- That basically the `Host` minus the routes.
- So `AppLayout` could be a good base for any web app, `Host` uses `AppLayout` and then we can also use `AppLayout` in tests..

and then I thought I needed to back away and have you bring me back down to earth so I left it

Reply:
It does hollow out the Host and moves a bunch of things into the common app which aren't reused elsewhere.
Aside from testing, I'm not seeing why there would be a need to have multiple AppBar per site or LH drawer etc.

unless, we are thinking to take the whole common bundle and share that across sites / apps. now that would be awesome! I'm thinking then we could have the health supply hub ( and IRIMS, universal codes..?? ) share design and components with open mSupply.

there is real potential there.. though to share the common we'd have to have it in a package, so there's a bit of work later on when we're ready for that too.

Reply:
Yup, having common become a shared package for all web apps is what I was thinking. I wasn't sure about previous projects, but definitely any future ones.

Yeah, having it in a package would be annoying.

`@openmsupply-client` -> `@openmsupply-web`  🤔

Reply:
yes!! the rename. I like `-web` much better